ceph-releases/luminous/ubuntu/16.04/daemon/entrypoint.sh.in

```diff
@@ -6,6 +6,22 @@ source variables_entrypoint.sh
 source common_functions.sh
 source debug.sh
 
+
+##################
+# PRE/POST TASKS #
+##################
+
+function sigterm_cleanup_post {
+  if [ "$CEPH_DAEMON" == 'OSD_CEPH_DISK_ACTIVATE' ]; then
+    local ceph_mnt
+    ceph_mnt=$(findmnt --nofsroot --noheadings --output SOURCE --submounts --target /var/lib/ceph/osd/ | tail -n +2)
+    for mnt in $ceph_mnt; do
+      log "Unmounting $mnt"
+      umount "$mnt"
```

Comment:
Can you log something if the umount fails ? That could be useful when debugging this issue.